pkg/asset/installconfig/openstack/validate.go

```diff
@@ -0,0 +1,38 @@
+package openstack
+
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/openshift/installer/pkg/asset/installconfig/openstack/validation"
+	"github.com/openshift/installer/pkg/types"
+)
+
+var (
+	validFlavor = "valid-flavor"
```

Comment:
These look like const for testing.. keep them in _test.go

Reply:
oh oops. good catch